packages/components/nodes/documentloaders/FireCrawl/FireCrawl.ts

```diff
@@ -304,60 +429,115 @@ class FireCrawl_DocumentLoaders implements INode {
+                description: 'Tags to include in the output',
+                optional: true,
+                additionalParams: true
+            },
+            {
+                // excludeTags
+                label: '[Scrape] Exclude Tags',
+                name: 'excludeTags',
+                type: 'string',
+                description: 'Tags to exclude from the output',
+                optional: true,
+                additionalParams: true
+            },
+            {
+                // onlyMainContent
+                label: '[Scrape] Only Main Content',
+                name: 'onlyMainContent',
+                type: 'boolean',
+                description: 'Extract only the main content of the page',
                 optional: true,
                 additionalParams: true
             },
+            {
+                // limit
+                label: '[Crawl] Limit',
+                name: 'limit',
+                type: 'string',
+                description: 'Maximum number of pages to crawl',
+                optional: true,
+                additionalParams: true,
+                default: '10000'
+            },
             {
                 // generateImgAltText
-                label: 'Generate Image Alt Text',
+                label: '[Crawl] Generate Image Alt Text',
                 name: 'generateImgAltText',
                 type: 'boolean',
                 description: 'Generate alt text for images',
                 optional: true,
                 additionalParams: true
             },
             {
-                // returnOnlyUrls
-                label: 'Return Only URLs',
-                name: 'returnOnlyUrls',
-                type: 'boolean',
-                description: 'Return only URLs of the crawled pages',
+                label: '[Extract] Schema',
+                name: 'extractSchema',
+                type: 'json',
+                description: 'JSON schema for data extraction',
                 optional: true,
                 additionalParams: true
             },
             {
-                // onlyMainContent
-                label: 'Only Main Content',
-                name: 'onlyMainContent',
-                type: 'boolean',
-                description: 'Extract only the main content of the page',
+                label: '[Extract] Prompt',
+                name: 'extractPrompt',
+                type: 'string',
+                description: 'Prompt for data extraction',
+                optional: true,
+                additionalParams: true
+            },
+            {
+                label: '[Extract] Job ID',
+                name: 'extractJobId',
+                type: 'string',
+                description: 'ID of the extract job',
                 optional: true,
                 additionalParams: true
             }
-            // ... (other input parameters)
         ]
         this.outputs = [
             {
```

Comment:
We can't change the output here, as they are meant to connect to other nodes:
![image](https://github.com/user-attachments/assets/4841058c-154e-4859-83c0-1b8b6c693a64)

Reply:
I will revert this part!